ktoml-core/src/commonMain/kotlin/com/akuleshov7/ktoml/writers/TomlEmitter.kt

```diff
@@ -223,7 +223,12 @@ public abstract class TomlEmitter(config: TomlOutputConfig) {
                 float.isNaN() -> "nan"
                 float.isInfinite() ->
                     if (float > 0) "inf" else "-inf"
-                else -> float.toString()
+                else -> {
```

Comment:
May be need to report/fix in Kotlin? What do you think? Or not a bug

Reply:
I think it is. I'm guessing something to do with JS's dynamic typing